selenium_tests/bulk_create/create_new_ci_tests.py

```diff
@@ -54,11 +54,10 @@ def test_create_site_without_short_title(self):
             self.test_data['course_code'],
             self.test_data['course_title'],
             '',  # simulates no short title entered in form
-            self.test_data['term'],
             expected_course_code_text)
 
     def _test_create_site(self, course_code, course_title, course_short_title,
-                          course_term, expected_course_code):
```

Comment:
Great point/call!   I initially removed it because it wasn't critical that a term had to be specified for new course creation (it would default to the first one in the dropdown). Also, we were running in "term 1  not found" errors. However, per our discussions, I investigated on making the terms work for these tests.  It turns out there is a  consistency issue in the code.  In the "create new site for existing course" term dropdown, the option value is "string:xxxx", while In the create new site for new course" term dropdown, the option value is "numbers:xxxx".   I've updated the test data in local.py for it can fetch the right term value, updated the code to include terms again, and verified that it works after these changes.